### DISCUSSION TITLE
Changing duration of helicopter events or manually ending them

### DISCUSSION THREAD

**Discussion by doctorthoss** [TEHE-Studios/ExpandedHelicopterEvents#140] [Q&A]
As server admin is it possible to shorten the duration of or force end a helicopter event? the supply drop event in particular seems to last a really long time if you don't go and get the drop (which I usually can't do because the HUD icon disappears too quick), and doesn't even seem to draw many zombies anyway.

In my server right now there's a plane or helicopter still flying around 24 in-game hours after dropping the crate. And this is with day length set to 2 IRL hours so it's super annoying lmao

**Comment by Chuckleberry-Finn** [comment]
This is a bug, the sound is getting stuck.

**Reply by doctorthoss** [reply]
i'll try rebooting the server, it's been going 72 in game  hours now

**Reply by Chuckleberry-Finn** [reply]
Reconnecting should fix the issue for each player respectively, if you are reloading to find the event still there let me know.

Reboot would fix the issue as well.
